Mars Onboarding/StepDefinitions/LanguagesAndSkillsStepDefinitions.cs

```diff
@@ -0,0 +1,201 @@
+using System;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using Reqnroll;
+using Mars_Onboarding.Pages;
+using Mars_Onboarding.Support;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace Mars_Onboarding.StepDefinitions
+{
+    [Binding]
+    public class LanguagesAndSkillsStepDefinitions : CommonDriver
+    {
+        public new static IWebDriver driver;
+        public LanguagesAndSkillsStepDefinitions()
+        {
+            driver = new ChromeDriver();
+        }
+
+
+        [Given("I logged in successfully")]
+        public void GivenILoggedInSuccessfully()
+        {
+            LoginPage loginPage = new LoginPage();
```

Comment:
you should only create the object once within the constructor this way you dont need to do LanguagesPage languagesPage = new LanguagesPage(); in every step